tests/appsec/rules-suspicious-requests-response.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "2.2",
+  "rules": [
+    {
+      "id": "tst-037-005",
+      "name": "Test block on response code",
+      "tags": {
+        "type": "lfi",
+        "crs_id": "000005",
+        "category": "attack_attempt"
+      },
+      "conditions": [
+        {
+          "parameters": {
+            "inputs": [
+              {
+                "address": "server.response.status"
+              }
+            ],
+            "regex": "404"
+          },
+          "operator": "match_regex"
+        }
+      ],
+      "transformers": [],
+      "on_match": [
+        "block"
+      ]
+    }
+
+  ]
+}
```

Comment:
very minor nit: can we add new lines to these json files

Reply:
I used automatic reformat tool on all the json files here.